pages/api/sidebar/documents/[id]/publish.js

```diff
@@ -87,6 +87,14 @@ export default async function Handler(req, res) {
       articleData['content'] = processedData['formattedElements'];
       articleData['main_image'] = processedData['mainImage'];
 
+      delete articleData.first_published_at;
+      delete articleData.last_published_at;
+      delete articleData.article_tags;
+      delete articleData.article_authors;
```

Comment:
Ah - we're deleting the article tags and authors from the articleData because those are handled separately, and we otherwise send the articleData to Hasura as variables in the insert/upsert article mutations. The publish dates being set in the database at the trigger function level, we also don't want to pass those values to the mutation.

(passing values that aren't defined in a mutation or query in graphql will throw an error saying something like "unexpected variables found")

Reply:
Right! Okay.